NatureArea.py

```diff
@@ -0,0 +1,245 @@
+from WikidataItem import WikidataItem
+import importer_utils as utils
+
+
+class NatureArea(WikidataItem):
+    """
+    Extension of WikidataItem to create nature area objects.
+
+    Handles both reserves and national parks.
+    """
+
+    def generate_ref_url(self):
+        url = "http://nvpub.vic-metria.nu/naturvardsregistret/rest/omrade/{}/G%C3%A4llande"
+        return url.format(self.raw_data["NVRID"])
+
+    def create_sources(self):
+        """Create the references for all statements."""
+        self.sources = {}
+        item_nr = self.items["source_nr"]
+        item_np = self.items["source_np"]
+        url = self.generate_ref_url()
+        self.sources["np"] = self.make_stated_in_ref(item_np,
+                                                     "2015-12-18",
+                                                     url)
+        self.sources["nr"] = self.make_stated_in_ref(item_nr,
+                                                     "2015-12-18",
+                                                     url)
+
+    def set_labels(self):
+        """
+        Add labels, optionally in multiple languages.
+
+        A number of items have the word natur/domän etc. reservat
+        included in the name. In this case, the name is
+        added only in Swedish. Otherwise, if it's a pure
+        geographical name, it is added in a number of Latin script
+        languages.
+        """
+        swedish_name = self.raw_data["NAMN"]
+        if ("reservat" in swedish_name.lower() or
+                "nationalpark" in swedish_name.lower() or
+                "skärgård" in swedish_name.lower()):
+            languages = ["sv"]
+        else:
+            languages = [
+                "sv", "en", "da", "fi", "fr", "pt",
+                "pl", "nb", "nn", "nl", "de", "es"
+            ]
+        for language in languages:
+            self.add_label(language, swedish_name)
+
+    def set_descriptions(self):
+        """
+        Add descriptions in various languages.
+
+        National park format: "national park in Sweden".
+        Reserve format: "nature reserve in <län>, Sweden".
+        """
+        if self.raw_data["SKYDDSTYP"] == "Nationalpark":
+            np_dictionary = self.glossary["np_description"]
+            for language in np_dictionary:
+                description = np_dictionary[language]
+                self.add_description(language, description)
+        elif self.raw_data["SKYDDSTYP"] == "Naturreservat":
+            county_without_lan = self.raw_data["LAN"].split(" ")[:-1]
+            county_name = " ".join(county_without_lan)
+            if utils.get_last_char(county_name) == "s":
+                county_name = county_name[:-1]
+
+            nr_dictionary = self.glossary["nr_description"]
+            fi_locations = self.glossary["location_in"]["fi"]
```

Comment:
Because it sounds better? You can do it both ways, but I think I see läänissä more often.